augur/merge.py

```diff
@@ -503,3 +579,85 @@ def quote(s):
     # …and instead quote a final empty string down here if we're still empty
     # after joining all our parts together.
     return quoted if quoted else shquote('')
+
+
+def run_command(command: str, print_stdout: bool):
+    """Run a command, clearly marking any messages that resemble SeqKit output."""
+    process = subprocess.Popen(
+        ['bash', '-euo', 'pipefail', '-c', command],
+        stdout=subprocess.PIPE if print_stdout else subprocess.DEVNULL,
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+
+    if print_stdout:
+        for line in iter(process.stdout.readline, ''):
+            print(line.rstrip())
+
+    for line in iter(process.stderr.readline, ''):
+        # Remove ANSI escape sequences common in SeqKit output.
+        line = re.sub(r'\x1b\[[\d;]+m', '', line)
+
+        # Detect messages from seqkit and append a prefix.
+        if line.startswith("[INFO]"):
+            print_debug(f"[SeqKit] {line.rstrip()}")
+        elif line.startswith("[ERRO]"):
+            print_info(f"[SeqKit] {line.rstrip()}")
+        else:
+            print_info(line.rstrip())
+
+    # Wait for the seqkit process to finish.
+    # FIXME: address potential deadlock issue?
+    # <https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait>
+    # I think this is safe since process.stdout and process.stderr are iterated manually.
+    return process.wait()
+
+
+def seqkit():
+    """
+    Internal helper for invoking ``seqkit``, the SeqKit CLI program.
+
+    Unlike ``sqlite3()``, this function is not a wrapper around subprocess.run.
+    It is meant to be called without any parameters and only returns the
+    location of the executable. This is due to differences in the way the two
+    programs are invoked.
+    """
+    seqkit = shquote(os.environ.get("SEQKIT", which("seqkit")))
+    if not seqkit:
+        raise AugurError(dedent(f"""\
+            Unable to find the program `seqkit`.  Is it installed?
+            In order to use `augur merge`, the SeqKit CLI must be installed
```

Comment:
_non-blocking_

I don't think there's anything but a CLI for SeqKit?

```suggestion
            In order to use `augur merge`, SeqKit must be installed
```

Reply:
Fixed:

https://github.com/nextstrain/augur/blob/e34a62bdc43a1e12cd8b58277f750a27bb2e20e7/augur/merge.py#L601